goth/runner/cli/yagna_payment_cmd.py

```diff
@@ -72,19 +68,17 @@ def from_dict(source: dict):
 class YagnaPaymentMixin:
     """A mixin class that adds support for `<yagna-cmd> payment` commands."""
 
-    def payment_fund(
-        self: CommandRunner, payment_driver: PaymentDriver = DEFAULT_PAYMENT_DRIVER
-    ) -> None:
+    def payment_fund(self: CommandRunner, payment_driver: str) -> None:
```

Comment:
Perhaps instead of taking a `payment_driver` string argument this should accept a `PaymentConfig` object? I think this would be slightly easier to use from the API's end user's perspective.